src/routines/routines.ycp

```diff
@@ -640,7 +641,10 @@ define list<map> ReadHardware(string hwtype) {
     /* Confirmation: label text (detecting hardware: xxx) */
     string hwstring = _("All Network Devices");
     if(haskey(hwstrings, hwtype)) hwstring = hwstrings[hwtype]:"";
-    if(!Confirm::Detection(hwstring, "yast-lan")) return [];
+
+    // don't ask in the 1st stage, if we need to save a netcard config then network was already
+    // used for installation and asking is useless
```

Comment:
I just tried the latest DVD SP4 build on x86_64 with `manual=1` and the only confirmation displayed in the first stage was the USB driver load confirmation and no network config was saved at the end of the first stage.

I guess the s390s are usually installed from network and therefore there is a network detection and the config is then saved.